backend/src/jobs/querySupabase.ts

```diff
@@ -0,0 +1,23 @@
+// jobs/querySupabase.js
+const dotenv = require("dotenv");
+const path = require("path");
+dotenv.config({ path: path.resolve(__dirname, "../../../.env") });
+const { createClient } = require("@supabase/supabase-js");
+
+const supabase = createClient(
```

Comment:
You were already creating a client in another file. You already have a repository folder, any db calls here should reference your repository. Let's move any and all references to DB operations to a file within repository.